### DISCUSSION TITLE
how to make wrangler use yarn instead of npm?

### DISCUSSION THREAD

**Discussion by weiklr** [cloudflare/wrangler-action#177] [Q&A]
Hi, 
I noticed that wrangler uses npm out of the box, but my github actions workflow is using yarn. Is there some flag that i can set to make wrangler use yarn instead?

I do notice there's a 'packageManager' variable in the source code. can that be used for this purpose?

**Comment by Hugo-C** [comment]
README now have a [paragraph](https://github.com/cloudflare/wrangler-action?tab=readme-ov-file#using-a-different-package-manager) about it.
